build/build.c

```diff
@@ -303,6 +300,24 @@ static rpmRC doCheckBuildRequires(rpmts ts, rpmSpec spec, int test)
     return rc;
 }
 
+static rpmRC doBuildDir(rpmSpec spec, int test, StringBuf *sbp)
+{
+    char *doDir = rstrscat(NULL,
+			   "rm -rf ", spec->buildDir, "\n",
```

Comment:
We probably want to quote the argument to rm -rf, just in case 🥶 😄

Reply:
Oops, yep

Reply:
Quotes added now.

Reply:
Seems like this has reoccurred :smile:

Reply:
Don't forget about this one :smile:

Reply:
Aagh :laughing:

Reply:
Fixed now, and thanks for the reminder. Had totally forgotten it...